libs/examples/src/table/table-favorite-column-no-header-example/table-favorite-column-no-header-example.html

```diff
@@ -2,7 +2,7 @@
   <dt-favorite-column
     name="favorite"
     sortable="false"
-    (change)="doSomething($event)"
+    (change)="doSomething()"
```

Comment:
```suggestion
    (change)="doSomething($event)"
```

Reply:
`doSometing()` doesn't have a parameter anymore which results in the build step failing
I'll extend the function with that parameter